### ISSUE TITLE
new RealmConfiguration.Builder(appContext) can not be accessed from outside package

### ISSUE THREAD

**Issue by SammyO** [Zhuinden/realm-book-example#9]
Using this code as-is creates an error on `new RealmConfiguration.Builder(appContext)`, as this method isn't public and can't be accessed from outside the Realm package.
Propose to replace it with:
```
Realm.init(appContext);
setRealmConfiguration(new RealmConfiguration.Builder().initialData(new RealmInitialData())
                    .deleteRealmIfMigrationNeeded()
                    .build());
```

**Comment by Zhuinden** [comment]
The code in this example is Realm 1.2.0, there is an issue tracking the update to Realm 3.1.1+. https://github.com/Zhuinden/realm-book-example/issues/8 

I'll do that only when all relevant issues mentioned in that issue have been fixed, and so is the JNI error introduced by a change in 3.1.0

**Comment by Zhuinden** [comment]
Fixed by updating to Realm-Java 3.1.1
